Code/Framework/AzTest/AzTest/Utils.h

```diff
@@ -20,71 +20,90 @@ namespace AZ::IO
 namespace AZ
 {
     class SettingsRegistryInterface;
-    namespace Test
+}
+
+namespace AZ::Test
+{
+    /*! Command line parameter functions */
+    //! Check to see if the parameters includes the specified parameter
+    bool ContainsParameter(int argc, char** argv, const std::string& param);
```

Comment:
Maybe `GetParameterList(...) < 0` is implied and we don't need an extra function here?

Reply:
I can update this function to return `GetParameterIndex() >=0;`